### PR TITLE
Add Python version of step-53

### PR DESCRIPTION
**Pull request by agrayver**
Part of #9015

### CODE REVIEW

**Comment by agrayver** [comment]
Here is the direct [link](https://github.com/dealii/dealii/blob/c8a1d051a86e3a29a4bd723a8fa6e48cca63a592/examples/step-53/step-53.ipynb) to the rendered notebook.

File: contrib/python-bindings/notebooks/index.ipynb [hunk 1]
```diff
@@ -13,10 +13,20 @@
...
     "[step-62](https://github.com/dealii/dealii/tree/master/examples/step-62/step-62.ipynb) shows how to calculate the [energy band gap](https://en.wikipedia.org/wiki/Band_gap) and the\n",
     "mechanical resonance of a [micropillar superlattice cavity](https://doi.org/10.1103/PhysRevA.94.033813).\n"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
```
[the remark this answers is not in the document]
Comment (agrayver) [reply contrib/python-bindings/notebooks/index.ipynb:29]: I changed the notebook and executed it. I don't know why jupyter puts it, but I think this is harmless.

**Comment by agrayver** [comment]
@Rombur I have addressed your comments, thanks.

File: contrib/python-bindings/notebooks/index.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (Rombur) [reply contrib/python-bindings/notebooks/index.ipynb:29]: This adds a new empty cell. Just remove the code.

File: examples/step-53/step-53.ipynb [hunk 2]
```diff
@@ -0,0 +1,732 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "module_path = os.path.abspath('/home/agrayver/lib/dealii/build_gofem/lib/python3.7/site-packages')\n",
```
Comment (Rombur) [inline examples/step-53/step-53.ipynb:51]: Maybe use a generic path here instead.

**Comment by agrayver** [comment]
Fixed and squashed.

**Comment by Rombur** [comment]
Hmm there is a failing test (looks unrelated to this PR). Can you rebase on master to see if that fixes the problem.

**Pull request merged**